pkg/resources/management.cattle.io/v3/project/validator.go

```diff
@@ -24,6 +27,7 @@ const (
 	projectQuotaField   = "resourceQuota"
 	clusterNameField    = "clusterName"
 	namespaceQuotaField = "namespaceDefaultResourceQuota"
+	containerLimitField = "namespaceDefaultResourceQuota"
```

Comment:
Is this accurate? I think this should be changed to:
```suggestion
	containerLimitField = "containerDefaultResourceLimit"
```

Let me know if I am misunderstanding or have something wrong there.

Reply:
Nope, it's just my oversight.